.changelog/2413.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+cli: Update base commands to default to all apps within the project
```

Comment:
```suggestion
cli: Update base commands to default to all apps within the project if project has more than one application
```

I think in the single app case this is still true, but the big change is now if your project has multiple apps you're no longer required to specify an app flag.